icu4c/source/i18n/timezone.cpp

```diff
@@ -527,6 +527,11 @@ TimeZone::detectHostTimeZone()
 
 // -------------------------------------
 
+static UMutex *gMutex() {
```

Comment:
Even though gMutex is a file-static, how about a more descriptive name?

Reply:
I agree with choosing a more distinctive name.

Reply:
changed to gDefaultZoneMutex